homeassistant/components/knocki/quality_scale.yaml

```
@@ -53,7 +53,7 @@ rules:
   discovery-update-info:
     status: exempt
     comment: This is a cloud service and does not benefit from device updates.
```

```suggestion
  discovery-update-info: done
```
we update all we can

---

We don't update connection info though